app/src/main/java/com/illiouchine/jm/ui/screen/PollSetupScreen.kt

```diff
@@ -219,7 +219,7 @@ fun PollSetupScreen(
                 enabled = pollSetupState.pollSetup.proposals.size > 1,
                 onClick = {
                     // Rule: if the poll's subject was not provided, use a default.
-                    if (subject.isBlank()) {
+                    if (subject.isEmpty() || subject.isBlank()) {
```

Comment:
Useless, I know.  We'll remove this once we've fixed that subject generation business.